lms/djangoapps/ccx/views.py

```diff
@@ -141,10 +144,17 @@ def dashboard(request, course, ccx=None):
         schedule = get_ccx_schedule(course, ccx)
         grading_policy = get_override_for_ccx(
             ccx, course, 'grading_policy', course.grading_policy)
+        enrolled_students = CourseEnrollment.objects.filter(course_id=ccx_locator, is_active=True)
         context['schedule'] = json.dumps(schedule, indent=4)
         context['save_url'] = reverse(
             'save_ccx', kwargs={'course_id': ccx_locator})
-        context['ccx_members'] = CourseEnrollment.objects.filter(course_id=ccx_locator, is_active=True)
+        context['ccx_members'] = enrolled_students \
```

Comment:
What do you think about the following to improve readability?
```suggestion
        context['ccx_members'] = (
    enrolled_students
    if not configuration_helpers.get_value(
        'HIDE_MASTER_COURSE_STAFF_FROM_STUDENT_LIST', False)
    else exclude_master_course_staff_users(
        users=enrolled_students,
        course_key=ccx_locator,
        model='CourseEnrollment',)             
)

Reply:
Addressed. @diegomillan